java/openjdk8/Portfile

```diff
@@ -50,6 +50,7 @@ depends_build       port:autoconf \
                     port:gmake \
                     port:pkgconfig \
                     port:bash
+depends_run         port:libiconv
```

Comment:
Should this not be depends_lib?

Reply:
According to other people responding to https://trac.macports.org/ticket/68278 `depends_run` would be ok, but I don't know if the library just needs to be present at runtime, or if its presence has an effect on the build process.

Reply:
Normally a library has to be present in order to link with it. Some of the build-time dependencies depend on libiconv, so it would already have been present when building, just not deliberately.

Reply:
Ok, I've created https://github.com/macports/macports-ports/pull/22261 for `openjdk8`. Will do the same for the other major versions later.